mantidimaging/core/utility/memory_usage.py

```diff
@@ -28,22 +28,25 @@ def mb(self):
     return Value(meminfo.available - meminfo.total * MEMORY_CAP_PERCENTAGE)
 
 
-def get_memory_usage_linux(kb=False, mb=False):
+def get_memory_usage_linux(kb=False, mb=False) -> tuple[float, float]:
     """
+    Get the memory usage of the system in bytes, kilobytes or megabytes.
+
     :param kb: Return the value in Kilobytes
     :param mb: Return the value in Megabytes
     """
     import psutil
 
     meminfo = psutil.virtual_memory()
-    tuple_to_return = ()  # start with empty tuple
     # meminfo.used gives the size in bytes
-    if kb:
-        tuple_to_return += (meminfo.used / 1024, )
-
-    if mb:
-        tuple_to_return += (meminfo.used / 1024 / 1024, )
-    return tuple_to_return
+    if kb and mb:
+        return (meminfo.used / 1024, meminfo.used / 1024 / 1024)
+    elif kb:
+        return (meminfo.used / 1024, 0.0)
+    elif mb:
+        return (0.0, meminfo.used / 1024 / 1024)
+    else:
+        return (0.0, 0.0)
```

Comment:
I think the original function here is over complex. Maybe it should just return a value in bytes, and the place using it can convert to kb or mb as needed.

Reply:
Seems to only be called in one place in the same class by `get_memory_usage_linux_str()`so I've simplified to only return bytes and allow `get_memory_usage_linux_str()` to convert into kb and mb